project/scripts/examples/bisect-cli-example.sh

```diff
@@ -0,0 +1,3 @@
+#!/usr/bin/env bash
+
+scala-cli compile -S "$1" file1.scala file2.scala
```

Comment:
Using this validation script will not work because it will disappear if the bisect goes before the commit where it was created.

Reply:
This is why I moved was going to move the validation into the `git bisect` in #16347.

Reply:
Maybe we can add these to the documentation. This way users will copy them into a new file that will not be affected by `git bisect`.

Reply:
This has been already said in the usage message in `bisect.scala`. But just in case I added comments in the example scripts.